example/README.md

```diff
@@ -0,0 +1,16 @@
+# example
+
+A new Flutter project.
+
+## Getting Started
+
+This project is a starting point for a Flutter application.
+
+A few resources to get you started if this is your first Flutter project:
+
+- [Lab: Write your first Flutter app](https://docs.flutter.dev/get-started/codelab)
+- [Cookbook: Useful Flutter samples](https://docs.flutter.dev/cookbook)
+
+For help getting started with Flutter development, view the
+[online documentation](https://docs.flutter.dev/), which offers tutorials,
+samples, guidance on mobile development, and a full API reference.
```

Comment:
can include instructions to build the example/demo for different target platform
for example:
- flutter build -android

Reply:
can ignore all build folders as they are auto generated after the build cmd

Reply:
instructions to create a demo app with the example/lib/main.dart given in this github